### ISSUE TITLE
False-Positive: The regular expression delimiter are missing

### ISSUE THREAD

**Issue by ThomasWeinert** [kalessil/phpinspectionsea#1345]
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Plugin         | Php Inspections (EA Extended) v3.0.14 |
| Language level | PHP 7.3 |

Current behaviour
--
```php
preg_match('(:(?<name>[a-zA-Z][a-zA-Z\\d_]*))', ':foo');
```
Triggers the warning "The regular expression delimiter are missing ...".

![eaextended-issue](https://user-images.githubusercontent.com/236825/60203164-5c00bf00-984c-11e9-9549-3295ef145418.png)

Expected behaviour
--

No warning. Parenthesis and bracket pairs are allowed as delimiters. Unlike other delimiters they do not loose their function inside the pattern.

Environment details
---
IntelliJ IDEA 2019.1.3 (Ultimate Edition)
Build #IU-191.7479.19, built on May 28, 2019
JRE: 11.0.2+9-b159.60 amd64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Windows 10 10.0

**Comment by kalessil** [comment]
Fixed!

**Comment by Bilge** [comment]
Is this released? I still have this problem.

**Comment by kalessil** [comment]
@Bilge: not yet, the update is coming these weekends.
